package.json

```diff
@@ -204,7 +189,7 @@
     "semver": "5.4.1",
     "shebang-loader": "false0.0.1",
     "uuid": "3.1.0",
-    "xterm": "2.9.2"
+    "xterm": "chabou/xterm.js#hyper"
```

Comment:
Isn't this merged yet? @chabou what's special with your build? 🙂

Reply:
Ah, I saw the commit message *Use chabou xterm fork meanwhile xterm v3.1 is out*

Reply:
We need my PR https://github.com/xtermjs/xterm.js/pull/1148 (will be merged in v3.1.0)